libs/coin-modules/coin-filecoin/src/bridge/utils.ts

```diff
@@ -0,0 +1,80 @@
+import { AccountLike } from "@ledgerhq/types-live";
+import { BigNumber } from "bignumber.js";
+import { getEquivalentAddress } from "../network";
+
+export enum Methods {
+  Transfer = 0,
+  ERC20Transfer = 1,
+  InvokeEVM = 3844450837,
+}
+
+export enum AccountType {
+  Account = "Account",
+  TokenAccount = "TokenAccount",
+}
+
+export enum BotScenario {
+  DEFAULT = "default",
+  ETH_RECIPIENT = "eth-recipient",
+  F4_RECIPIENT = "f4-recipient",
+  TOKEN_TRANSFER = "token-transfer",
+}
+
+const validHexRegExp = new RegExp(/^(0x)?[a-fA-F0-9]+$/);
+const validBase64RegExp = new RegExp(
+  /^(?:[A-Za-z\d+/]{4})*(?:[A-Za-z\d+/]{3}=|[A-Za-z\d+/]{2}==)?$/,
+);
+
+// TODO Filecoin - Use the new package @zondax/ledger-utils instead
+export const isNoErrorReturnCode = (code: number) => code === 0x9000;
```

Comment:
if it's something related to device only, maybe it should not be in coin-filecoin

Reply:
If I don't put it in `utils.ts`, where should I put it?  Because it’s used in another function in my `utils.ts`.

Reply:
done